app/src/main/AndroidManifest.xml

```diff
@@ -106,7 +106,7 @@
             android:exported="false"
             android:label="@string/support_title"
             android:parentActivityName=".ui.home.HomeActivity"
-            android:theme="@style/Theme.AdAway.NoActionBar.Red" />
+            android:theme="@style/Theme.AdAway" />
```

Comment:
Is this needed if you change it to the default theme?

Reply:
Oh true, that's not needed anymore, silly me.